Dockerfile

```diff
@@ -150,12 +150,13 @@ ENV IS_DOCKER=true
 ENV SDK_SRC=/src
 ENV OUTPUT_DIR=/out
 ENV SDK_REVISION=
+Env SDK_REPO=
```

Comment:
This is the first time I've seen a non-capitalised Docker command.  Interesting.

Reply:
Resolved in 41242a65e6bbd5660c7df1262e05ec069b09e022